packages/Tag/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@paprika/tag",
+  "version": "1.0.0",
+  "description": "The Tag component is for showing metadata like status, type, or levels.",
```

Comment:
This is a bit narrow... from the UX pitch:

> Tag can be both static UI labels or removable components. Static Tag is mostly used to show an item’s status and removable Tag will be used to add or group objects. 

The grammar can be touched up a little, and if you can condense it a little more, that would be even better.